test/Test/P/List.hs

```diff
@@ -19,6 +19,10 @@ prop_lastMaybe_empty :: Property
 prop_lastMaybe_empty =
   lastMaybe [] === (Nothing :: Maybe ())
 
+prop_count :: [Int] -> Property
+prop_count list =
+  count (>= 0) list === length (filter (>= 0) list) 
+
```

Comment:
Also should be able to get some `a -> Bool` functions from `CoArbitrary` so should be able to let it take a generated predicate?